### PR TITLE
feat(elements): add example primitives page with OTP example

### PR DESCRIPTION
**Pull request by alexcarpenter**
https://clerk.com/docs/pr/979/elements/examples/primitives

### CODE REVIEW

File: docs/elements/examples/primitives.mdx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+                        layoutId="otp-input-focus"
+                        transition={{ ease: [0.2, 0.4, 0, 1], duration: 0.2 }}
+                        className="absolute inset-0 z-10 rounded-[inherit] border border-sky-400 bg-sky-400/10 shadow-[0_0_8px_2px_theme(colors.sky.400/30%)]"
+                      />
+                    )}
+                  </div>
+                )}
+              />
+              <Clerk.FieldError className="block text-sm text-red-400" />
+            </Clerk.Field>
```
Comment (BRKalow) [inline docs/elements/examples/primitives.mdx:91]: Can we only show this in the code block? Everything else is just boilerplate I'd say

Comment (BRKalow) [reply docs/elements/examples/primitives.mdx:91]: Yep, exactly

Comment (alexcarpenter) [reply docs/elements/examples/primitives.mdx]: 👍🏼 https://github.com/clerk/clerk-docs/pull/979/commits/ba409bc7195bb83e93c1fc64861e491c9f4f8f2b

File: docs/elements/examples/primitives.mdx [hunk 2]
```diff
@@ -0,0 +1,54 @@
+---
+description: Examples of pre-built flows with Clerk Elements primitives.
```
Comment (alexcarpenter) [inline docs/elements/examples/primitives.mdx:2]:
```suggestion
description: Example implementations using Clerk Elements primitives.
```

Thoughts on description here?

File: docs/elements/examples/primitives.mdx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    style: {
+      height: 500
+    }
+  }
+]}>
+```tsx {{ title: 'OTP Input', collapsible: true }}
+<Clerk.Input
+  type="otp"
+  required
+  className="flex justify-center gap-x-1"
```
Comment (PixelJanitor) [inline docs/elements/examples/primitives.mdx:24]: Personally I prefer less typing so `gap-1` for me here but not a blocker if you prefer explicitly using x and y with gap

**Review by PixelJanitor — APPROVED** [review]
Amazing amazing amazing

**Pull request merged**